docs/Design/SRS/SRS.tex

```diff
@@ -662,6 +662,8 @@ \subsubsection{Assumptions} \label{sec_assumpt}
 
   \item [A:Continuum:\phantomsection\label{A:Continuum}]{The underlying molecular structure of matter is not considered and gaps and empty spaces within a material particle are ignored. The material is assumed to be continuous. \citep[p.\ 33--34]{Long1961}, \citep[p.\ 1-2]{Malvern1969} (RefBy: ?.)}
   
+  \item [A:Effective-Norm-Stress-Large:\phantomsection\label{A:Effective-Norm-Stress-Large}]{The effective normal stress is large enough that the shear strength to effective normal stress relationship can be approximated as a linear relationship. (RefBy: \hyperref[TM:Equilibrium]{TM:Equilibrium}))}
```

Comment:
@EmilSoleymani, you should remove this assumption, since we don't have any shear strength discussion in the settlement analysis.  I'm assuming you got this from the SSP example, which means that you found a mistake in the SSP example.  This assumption isn't related to the equilibrium model, but to another theory.